ui/src/App.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import {injectIntl} from 'react-intl';
+import {BrowserRouter, Route} from "react-router-dom";
+
+import Create from "./pages/Create.js";
+import GlobalHome from "./pages/GlobalHome.js";
+import RepoHome from "./pages/RepoHome.js";
+import Results from "./pages/Results.js";
+import View from "./pages/View.js";
+
+import './css/all.scss';
+
+const App = () => (
+  <BrowserRouter>
+    <div>
+      <Route exact path="/" component={GlobalHome} />
```

Comment:
You don't need to support those initially, but FYI. May be worth leaving TODO. We should support these URLs for backward compatibility:

* /global/home.html (had been the canonical URL of the global home page for long)
* /:repoId/ (with a trailing slash)

Reply:
Added a todo for global/home.html. Trailing slashes already work; I guess React Router just handles that for you.